crates/cli/src/commands/install.rs

```diff
@@ -105,7 +169,7 @@ mod tests {
         package_json.save().unwrap();
 
         let package_manager = PackageManager::new(&package_json_path).unwrap();
-        let args = InstallCommandArgs { prod: false, dev: true, no_optional: false };
+        let args = InstallCommandArgs { prod: false, dev: false, no_optional: false };
```

Comment:
@anonrig I did change this line for the sake of passing the tests. I don't know what is your original intent. I assumed that you just wanted to test if all the packages are installed and were interested in whether it filters the dependency group.

Reply:
Yes, this change is correct.